src/miner_sup.erl

```diff
@@ -105,7 +105,7 @@ init(_Args) ->
                                                    %% slot to handle this case.
                                                    GetPublicKey(KS+1)
                                            end;
-                                       {error, _} ->
```

Comment:
What about the catchall case?

Reply:
Assuming it can still throw other errors...

Reply:
Other errors will cause miner to crash.. And then restarted by the system. There's no good recovery or fallback at this stage